completion_aggregator/api/common.py

```diff
@@ -77,44 +85,20 @@ class CompletionViewMixin(object):
     """
     Common functionality for completion views.
     """
-
-    _allowed_requested_fields = {'mean', 'username'}
+    authentication_classes = (
+        JwtAuthentication,
+        OAuth2AuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser,
+    )
+    pagination_class = NamespacedPageNumberPagination
     permission_classes = (IsAuthenticated,)
+    _allowed_requested_fields = {'mean', 'username'}
     _effective_user = None
     _requested_user = None
 
     course_completion_serializer = None
     block_completion_serializer = None
 
-    @property
-    def authentication_classes(self):  # pragma: no cover
```

Comment:
I believe this can be kept as a method if renamed to [`get_authenticators()`](https://github.com/encode/django-rest-framework/blob/95d4843abeecea96754a147f4f2cca33e620ad09/rest_framework/views.py#L268-L272) (and not a `@property`)